code_generation/data/attribute_classes/output.json

```diff
@@ -125,6 +125,73 @@
                     "description": "deviation between the measured value and calculated value"
                 }
             ]
+        },
+        {
+            "name": "NodeShortCircuitOutput",
+            "base": "BaseOutput",
+            "is_template": true,
+            "attributes": [
+                {
+                    "data_type": "RealValue<sym>",
+                    "names": [
+                        "u_pu",
+                        "u",
+                        "u_angle"
+                    ],
+                    "description": "Initial short circuit voltage magnitude and angle"
+                }
+            ]
+        },
+        {
+            "name": "BranchShortCircuitOutput",
+            "base": "BaseOutput",
+            "is_template": true,
+            "attributes": [
+                {
+                    "data_type": "RealValue<sym>",
+                    "names": ["i_from"],
```

Comment:
I'd like to have single values not written as a list, so:
`"names": "i_from",`

Happen on more places ;)